test/utils/theme/css/tarallo.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+    font-family            : "Poppins Light";
+    src                    : url("Poppins-Light.ttf") format("truetype");

```

Comment:
I don't see this ttf-file included in this PR. Isn't it going to be needed?

Reply:
I removed it now since it was not included in the theme folder.